client/documents/strings.ts

```diff
@@ -7,5 +7,5 @@ import { __ } from '@wordpress/i18n';
 
 // Mapping of transaction types to display string.
 export const displayType = {
-	vat_invoice: __( 'VAT Invoice', 'woocommerce-payments' ),
+	vat_invoice: __( 'Tax Invoice', 'woocommerce-payments' ),
 };
```

Comment:
💭 I notice that this file is copy-pasta, presumably the transactions list implementation (see the code comment).

It's not necessarily in scope for this PR, but it might be worth removing this separate `strings.ts` file and defining this single string inline to reduce unnecessary import/export complexity.

Reply:
I think its being used to share the code between `/documents/filters` and `/documents/list`

Reply:
Ideally when we change the wording in the UI we'd change the code too – e.g. `vat_invoice` key in this little DRY string struct (if indeed it's worth an abstraction, as noted by Eric!). It makes for a bigger PR, but a more consistent code base, so IMO worth it.